server/src/main/java/org/bonitasoft/web/rest/server/api/resource/CommonResource.java

```diff
@@ -198,16 +199,18 @@ else if (t instanceof FileNotFoundException) {
                 LOGGER.log(Level.FINE, "***" + message);
             }
             getResponse().setStatus(Status.CLIENT_ERROR_NOT_FOUND);
+            errorMessage.setMessage("File Not Found");
         }
         else if (t instanceof NotFoundException) {
             if (LOGGER.isLoggable(Level.FINE)) {
                 LOGGER.log(Level.FINE, "***" + message);
             }
             getResponse().setStatus(Status.CLIENT_ERROR_NOT_FOUND);
+            errorMessage.setMessage("File Not Found");
```

Comment:
I don't think we should override the message here. This is an "engine" NotFoundException and not a FileNotFound exception.